HighwayPatrol.cpp

```diff
@@ -28,9 +30,9 @@ void HighwayPatrol::pullOver( Vehicle* v, bool willArrest, Highway* h )
     std::cout << name << ": vehicle is traveling " << v->speed - h->speedLimit << " miles per hour over the speed limit" << std::endl;
     if( willArrest )
     {
-        assert(false);
+        //assert(false);
         //print the vehicle type in this std::cout between "THE [" and "] PULL". 
-        std::cout << name << ": YOU IN THE [ " << " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
+        std::cout << name << ": YOU IN THE [ " << typeid(*v).name() << " ] PULL OVER AND SHOW YOUR HANDS" << std::endl;
```

Comment:
This is not the right way to get the name. 
use `if/else if` and casting to figure out the type and subsequent name to print.
- [ ] revise